packages/block-library/src/query/edit/inspector-controls/enhanced-pagination-control.js

```diff
@@ -1,58 +1,45 @@
 /**
  * WordPress dependencies
  */
-import { ToggleControl, Notice } from '@wordpress/components';
+import { ToggleControl } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { BlockTitle } from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
-import { useUnsupportedBlockList } from '../../utils';
+import { useUnsupportedBlocks } from '../../utils';
 
 export default function EnhancedPaginationControl( {
 	enhancedPagination,
 	setAttributes,
 	clientId,
 } ) {
-	const unsupported = useUnsupportedBlockList( clientId );
+	const { hasUnsupportedBlocks } = useUnsupportedBlocks( clientId );
+
+	let help = __( 'Browsing between pages requires a full page reload.' );
+	if ( enhancedPagination ) {
+		help = __(
+			"Browsing between pages won't require a full page reload, unless non-compatible blocks are detected."
+		);
+	} else if ( hasUnsupportedBlocks ) {
+		help = __(
+			"Force page reload can't be disabled because there are non-compatible blocks inside the Query block."
```

Comment:
Ignore this, I missed the renaming of the feature when I wrote it and forgot to go back and remove it.

Reply:
This text is actually fine because now that we inversed the control, the default (enabled) is to force a page reload, and what users can do is to _disable_ it.